packages/cli/src/nonInteractiveCli.ts

```diff
@@ -118,13 +118,35 @@ export async function runNonInteractive(
             prompt_id,
           };
 
+          if (config.getShowToolUsage()) {
+            console.error(
+              `[INFO] Using tool: ${fc.name} with args: ${JSON.stringify(
+                fc.args,
```

Comment:
I don't think we should log args, with current tools i don't think there are any PII but people can have custom tools that have PII or api keys, so probably should let the user know that we are using a tool and if something goes wrong show the error.